ios/NativeViews/IpfsImageView.h

```diff
@@ -0,0 +1,19 @@
+#import <UIKit/UIKit.h>
+
+#import <React/RCTResizeMode.h>
+
+@class RCTBridge;
+@class IpfsImageSource;
+
+@interface IpfsImageView : UIImageView
+
+- (instancetype)initWithBridge:(RCTBridge *)bridge NS_DESIGNATED_INITIALIZER;
```

Comment:
This is the Obj C native class that is exported to RN and backs `IpfsImage` JS class.